src/subscription_manager/repofile.py

```diff
@@ -60,7 +60,6 @@ class Repo(dict):
             'enabled': (1, '1'),
             'gpgcheck': (1, '1'),
             'gpgkey': (0, None),
-            'sslverify': (1, '1'),
```

Comment:
Why do you want to remove `sslverify` from `Repo` class? This class is used by many classes (not only `ZypperRepoFile`). Please do not do such changes that would modify behavior on other platforms.

It seems that `sslverify` is not supported by zypper. Current solution (deleting `sslverify` from `zypper_cont`) is IMHO decent.

Reply:
Sorry, i must have been blind and did not see that this setting is used elsewhere.